Packs/Zendesk/Integrations/Zendeskv2/README.md

```diff
@@ -271,7 +280,7 @@ Get organization's data.
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| organization_id | The ID of a specific organization. | Optional | 
+| organization_id | The ID of a specific organization. <br/>Required permissions: Admins, Agents.  | Optional | 
```

Comment:
Why do you add the `<br/>` ?  It's an HTML tag

Reply:
same